resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/internal/CircuitBreakerStateMachine.java

```diff
@@ -154,6 +156,14 @@ public String toString() {
         return String.format("CircuitBreaker '%s'", this.name);
     }
 
+    @Override
+    public void resetToClosedState() {
+        CircuitBreakerState previousState = stateReference.getAndUpdate(currentState -> new ClosedState(this));
+        if (previousState.getState() != CLOSED) {
+            publishStateTransitionEvent(StateTransition.transitionToClosedState(previousState.getState()));
```

Comment:
Should be `publishStateResetEvent`, or?

Reply:
Since it's a new event type, we should also add a new event handler `onStateReset` in `EventPublisher`.

Reply:
I found an *EmitterTest and changed that. Is the only place you wanted me to change?
Thanks for your CR